janus_core/calculations/geom_opt.py

```diff
@@ -2,17 +2,20 @@
 
 from logging import Logger
 from pathlib import Path
-from typing import Any, Callable, Optional
+from typing import Any, Callable, Optional, Union
 import warnings
 
 from ase import Atoms
 from ase.io import read, write
+import ase.optimize
 from ase.optimize import LBFGS
```

Comment:
```suggestion
```

Reply:
(Alternatively, `"LBFGS"` to look up the function, but then you lose IDE help etc. which makes setting extra kwargs more tricky, and I'm not sure what we gain, as we only check ase.optimize functions at the moment?)